cmd/nvidia-cdi-hook/chmod/chmod.go

```diff
@@ -113,15 +113,15 @@ func (m command) run(c *cli.Context, cfg *config) error {
 		return fmt.Errorf("failed to load container state: %v", err)
 	}
 
-	containerRoot, err := s.GetContainerRoot()
+	containerRoot, err := s.GetContainerRootDirPath()
```

Comment:
:heart:  Thanks for adding clarity here.

And now that I have read your explanation.. this is a path that is valid in the _host_ filesystem, and points to the container filesystem's root..? :)

Reply:
Yes. It is the absolute path on the host where the root of the container's filesystem is located.